### PR TITLE
Updated coupon card inputs

### PR DESCRIPTION
**Pull request by knivets**
It's a bit hacky, let me know if there's a better approach.

### CODE REVIEW

File: saas/api/coupons.py [hunk 1]
```diff
@@ -28,18 +28,49 @@
...
+    # plan with this slug already exists
+    slug = serializers.CharField(allow_blank=True, allow_null=True)
+
+    class Meta:
+        model = Plan
+        fields = ('slug', 'title')
+        read_only_fields = ['title']
+
 class CouponSerializer(serializers.ModelSerializer):
+    plan = PlanSerializer(required=False, allow_null=True)
```
[the remark this answers is not in the document]
Comment (knivets) [reply saas/api/coupons.py]: Not really possible without modifying the behavior of the other parts of the code that deal with plans.

File: saas/api/plans.py [hunk 2]
```diff
@@ -127,6 +127,10 @@ def post(self, request, *args, **kwargs):
 
     def get_queryset(self):
         self.queryset = self.organization.plans.all()
+        is_active = self.request.query_params.get('is_active')
```
Comment (knivets) [inline saas/api/plans.py]: I tried to use django-filter instead, but after wasting an hour I couldn't get it to work -- I tried a lot of combinations and I have no idea why it doesn't work.

Comment (smirolo) [reply saas/api/plans.py]: Using `query_params` is good. The logic seems convoluted though. I believe `self.queryset` is a class fields used as a default in the base class implementation of `get_queryset` (i.e. it shouldn't be modified).

**Comment by smirolo** [comment]
merged as a single commit 97a5e525.

**Pull request closed without merging**